Symbolic.fs

```diff
@@ -353,6 +353,33 @@ module Pretty =
     /// Pretty-prints a SMBoolExpr.
     let printSMBoolExpr = printBoolExpr (printSym printMarkedVar)
 
+/// Strip the marked part of the annotation
+/// and return just the internal 'var
+let unmarkMarkedVar =
+    function
+    | Before s            -> s
```

Comment:
That one is a bijection, this is a surjection that just pulls the 'var out of it, if i recall correctly.